docs/tutorials/AddRemark.md

```diff
@@ -340,23 +340,23 @@ save it with `Model#setPerson()`.
             throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
         }
 
-        Person personToEdit = lastShownList.get(index.getZeroBased());
-        Person editedPerson = new Person(personToEdit.getName(), personToEdit.getPhone(), personToEdit.getEmail(),
-                personToEdit.getAddress(), remark, personToEdit.getTags());
+        Person taskToEdit = lastShownList.get(index.getZeroBased());
```

Comment:
The class here is still not changed to Task

Reply:
same reply as above

Reply:
ok undid changes in these documents, please review again